dex/networks/polygon/genesis_config.go

```diff
@@ -0,0 +1,122 @@
+package polygon
+
+import (
+	"embed"
+	"encoding/json"
+	"fmt"
+	"math/big"
+
+	"decred.org/dcrdex/dex"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core"
+	"github.com/ethereum/go-ethereum/params"
+)
+
+// The allocs directory contains the genesis allocations for the various polygon
+// networks. See: https://github.com/maticnetwork/bor/tree/develop/core/allocs.
+
+//go:embed allocs
+var allocs embed.FS
+
+// These are the chain IDs of the various polygon network.
+const (
+	MainnetChainID = 137
+	TestnetChainID = 80001 // Mumbai
```

Comment:
Looks right for Mumbai based on https://www.alchemy.com/overviews/mumbai-testnet#how-to-get-started-using-the-mumbai-testnet, but why does Ankr say 137 for testnet?

`CORE[matic][RPC]: "https://rpc.ankr.com/polygon" reported wrong chain ID. expected 80001, got 137`

Reply:
Ankr is telling me that their Mumbai endpoint is https://rpc.ankr.com/polygon_mumbai

Reply:
> Ankr is telling me that their Mumbai endpoint is https://rpc.ankr.com/polygon_mumbai

Yes, you are correct. This is the correct Ankr Mumbai endpoint.

Reply:
Ah that would explain it.  Saw the error in your logs when you were testing Mumbai, but I see you switched the endpoint later: https://github.com/decred/dcrdex/pull/2399#issuecomment-1598202104
Ankr looks unreliable given all those deadline exceeded errors.

Reply:
> Ankr looks unreliable given all those deadline exceeded errors.

It could be because of my internet connection but ws is only available for premium accounts.